server/controllers/profile.js

```diff
@@ -38,4 +41,34 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
+});
+
+// @route POST /profile/upload
+// @desc Upload images to server
+// @access Private
+exports.uploadImages = asyncHandler(async (req, res) => {
```

Comment:
I would suggest renaming this controller to avoid confusion, we also have a util function to upload images to S3 😄